From 077185739fea98a8015aacdfab05a9174d074981 Mon Sep 17 00:00:00 2001
From: Prakash Brahmajyosyula <bprakash@marvell.com>
Date: Mon, 3 Dec 2018 11:26:29 +0300
Subject: [PATCH 0844/1051] soc: octeontx2: typo and cosmetic fixes.

This patch fixes couple of issues  print_npa_aura_ctx, one is swapped
prints for bp_ena, aura_drop, thresh_up, avg_level,thresh_qint_idx and
err_qint_idx. Other fixes such as extra tab/spaces.

Change-Id: I1caa8d5c6ea808bf9c2757b720d61baea2dc22f2
Signed-off-by: Prakash Brahmajyosyula <bprakash@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/1752
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2/rvu_debugfs.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/soc/marvell/octeontx2/rvu_debugfs.c b/drivers/soc/marvell/octeontx2/rvu_debugfs.c
index 264ada698e29..d87cd7f58194 100644
--- a/drivers/soc/marvell/octeontx2/rvu_debugfs.c
+++ b/drivers/soc/marvell/octeontx2/rvu_debugfs.c
@@ -270,9 +270,9 @@ static void print_npa_aura_ctx(struct npa_aq_enq_rsp *rsp)
 	pr_info("W1: pool drop ena\t%d\nW1: aura drop ena\t%d\n",
 		aura->pool_drop_ena, aura->aura_drop_ena);
 	pr_info("W1: bp_ena\t\t%d\nW1: aura drop\t\t%d\n",
-		aura->aura_drop, aura->shift);
+		aura->bp_ena, aura->aura_drop);
 	pr_info("W1: aura shift\t\t%d\nW1: avg_level\t\t%d\n",
-		aura->bp_ena, aura->avg_level);
+		aura->shift, aura->avg_level);
 
 	pr_info("W2: count\t\t%llu\nW2: nix0_bpid\t\t%d\nW2: nix1_bpid\t\t%d\n",
 		(u64)aura->count, aura->nix0_bpid, aura->nix1_bpid);
@@ -292,8 +292,8 @@ static void print_npa_aura_ctx(struct npa_aq_enq_rsp *rsp)
 	pr_info("W5: thresh_int\t\t%d\nW5: thresh_int_ena \t%d\n",
 		aura->thresh_int, aura->thresh_int_ena);
 	pr_info("W5: thresh_up\t\t%d\nW5: thresh_qint_idx\t%d\n",
-		aura->thresh_qint_idx, aura->err_qint_idx);
-	pr_info("W5: err_qint_idx \t\t%d\n", aura->thresh_up);
+		aura->thresh_up, aura->thresh_qint_idx);
+	pr_info("W5: err_qint_idx \t%d\n", aura->err_qint_idx);
 
 	pr_info("W6: thresh\t\t%llu\n", (u64)aura->thresh);
 }
-- 
2.17.1

